src/commands/deploy/genezio.ts

```diff
@@ -594,27 +622,30 @@ export async function functionToCloudInput(
 
     // copy everything to the temporary folder
     await fsExtra.copy(path.join(backendPath, functionElement.path), tmpFolderPath);
-    if (fsExtra.pathExistsSync(path.join(tmpFolderPath, "node_modules", ".pnpm"))) {
-        await fsExtra.remove(path.join(tmpFolderPath, "node_modules", ".pnpm"));
-        await fsExtra.copy(
-            path.join(backendPath, functionElement.path, "node_modules", ".pnpm"),
-            path.join(tmpFolderPath, "node_modules", ".pnpm"),
-            {
-                dereference: true,
-            },
-        );
+
+    // Handle JS/TS functions with pnpm
+    if (functionElement.language === "js" || functionElement.language === "ts") {
+        if (fsExtra.pathExistsSync(path.join(tmpFolderPath, "node_modules", ".pnpm"))) {
+            await fsExtra.remove(path.join(tmpFolderPath, "node_modules", ".pnpm"));
+            await fsExtra.copy(
```

Comment:
this is what i was talking about. we should do the same with the python packages